README.md

```diff
@@ -225,39 +225,48 @@ These tests are still somewhat far from good and you need to validate all result
 
 #### Service tests
 
-These tests are somewhat slow and fragile and because of that these are expected to be run manually.
+These tests are somewhat slow and fragile and because of that these are expect to be ran manually. To create files for your credentials, you need `jq` application installed, if you don't have it or you don't want to install it, you can create these files manually. The format is:
```

Comment:
`expected to be run` was correct, I would say

Reply:
Actually `expected to be manually run`. Fixed.

Reply:
It is still `expect to be manually run`. Should be `expected`.

Reply:
I was wondering why I changed those and after reverting to the original, I realized that grammar checker complained me about these, but I didn't check properly why (it was about using passive voice).  I changed it now quite a lot and actually changed most of this part of the README. Hope it is better now.

Reply:
Looks good, thanks!